### ISSUE TITLE
TypeError: tools.flat is not a function [BUG]

### ISSUE THREAD

**Issue by ms6rb** [FlowiseAI/Flowise#282]
**Describe the bug**
I'm getting this error ``TypeError: tools.flat is not a function`` in the chat on a fresh install from the quick start.

**To Reproduce**
1- Install from the **Quick Start**
2- Add OpenAI agent and MRKL agent for LLMs
3- Chat with bot
4- See  error in the chat ``TypeError: tools.flat is not a function``
5- See error in console:
```
TypeError: tools.flat is not a function
    at MRKLAgentLLM_Agents.init (C:\Users\Ms6RB\AppData\Roaming\npm\node_modules\flowise\node_modules\flowise-components\dist\nodes\agents\MRKLAgentLLM\MRKLAgentLLM.js:31:23)
    at buildLangchain (C:\Users\Ms6RB\AppData\Roaming\npm\node_modules\flowise\dist\utils\index.js:208:72)
    at async App.processPrediction (C:\Users\Ms6RB\AppData\Roaming\npm\node_modules\flowise\dist\index.js:539:44)
    at async C:\Users\Ms6RB\AppData\Roaming\npm\node_modules\flowise\dist\index.js:294:13
```
 
**Expected behavior**
I expected to get a response from the bot.

**Screenshots**
![image](https://github.com/FlowiseAI/Flowise/assets/33758164/b74e4c72-f172-4770-8111-b5161dbf7414)

**Setup**

-   OS: Windows 11
-   Node v19.9.0
-   npx\npm 9.6.5

**Comment by HenryHengZJ** [comment]
thanks! some of the browsers do not supports `.flat()` method. Put up a fix here - https://github.com/FlowiseAI/Flowise/pull/283

**Comment by ms6rb** [comment]
> thanks! some of the browsers do not supports `.flat()` method. Put up a fix here - #283

that solves the problem thanks <3
